medusa/providers/generic_provider.py

```diff
@@ -650,19 +660,17 @@ def _create_anime_search_string(self, show_scene_name, episode, search_string, a
         episode_string = show_scene_name + self.search_separator
 
         # If the show name is a season scene exception, we want to use the indexer episode number.
-        if (episode.scene_season > 1 and
+        if (episode.scene_season > 0 and
```

Comment:
I think my logic here was that there are only season scene exceptions from season 2?

Reply:
Yeah, from what I could see in the cache.db there are quite a few actually.

Reply:
Ow oops